.github/workflows/publish-stable.yml

```diff
@@ -51,7 +51,7 @@ jobs:
   npm:
     name: Publish to NPM & Github
     runs-on: buildjet-4vcpu-ubuntu-2204
-    needs: tag
+    needs: [tag, binaries]
```

Comment:
so the idea is that we only publish to npm if building binaries worked?

Reply:
I think it makes sense to have atomicity of the whole release process. The alternative is to publish the release evenso some binary creation fail.

Reply:
Make sense as binaries are really only published in the "create release" step, we should probably reorder the steps to match the order of execution